EvidenceApi.Tests/V1/Controllers/HealthCheckControllerTests.cs

```diff
@@ -32,7 +32,6 @@ public void ReturnsResponseWithStatus()
 
             response.Should().NotBeNull();
             response.StatusCode.Should().Be(200);
```

Comment:
the expected at line 30 can probably go if we're not going to compare anything to it